src/Components/CommentCard/CommentCard.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import './CommentCard.css';
+
+const CommentCard = ({ displayTextValue, value, sender, day }) => {
+
+	const displayText = [
+		`${value} points for you!`, 
+		`You get ${value} points!`, 
+		`Blam! ${value} points!`, 
+		`${value} points!  Yeah!`, 
+		`${value} big ole points!`
+	]
+
+	return (
+		<div className="comment-card-component">
+			<div className="card-header">
+				<h3>{displayText[displayTextValue]}</h3><h5 className="day-span">{day}</h5>
+			</div>
+			<p>Thanks for your help with JS array prototypes.  You totally crushed it!</p>
```

Comment:
is this placeholder text?

Reply:
the <p>?  yeah, just until we have messages coming thru